ActiveLabel/ActiveLabel.swift

```diff
@@ -213,10 +210,10 @@ typealias ElementTuple = (range: NSRange, element: ActiveElement, type: ActiveTy
             case .hashtag(let hashtag): didTapHashtag(hashtag)
             case .url(let originalURL, _): didTapStringURL(originalURL)
             case .custom(let element): didTap(element, for: selectedElement.type)
+            case .range(let id): rangeTapHandler?(id)
```

Comment:
Use the same approach as the options above where you notify vía delegate if there is not tap handler